archive/p/python/palindrome.py

```diff
@@ -0,0 +1,8 @@
+string = str(input("Enter the string:"))
```

Comment:
Thanks for moving this file! I didn't realize until now that it was sitting in root.